README.md

```diff
@@ -28,15 +28,14 @@ from seldon_deploy_sdk.auth import OIDCAuthenticator
 config = Configuration()
 config.host = "http://X.X.X.X/seldon-deploy/api/v1alpha1"
 config.oidc_client_id = "sd-api"
+config.oidc_client_secret = "sd-api-secret"
 config.oidc_server = "http://X.X.X.X/auth/realms/deploy-realm"
-config.username = "*******"
-config.password = "*******"
-config.auth_method = 'password_grant'
+config.auth_method = "auth_code"
 
 auth = OIDCAuthenticator(config)
 config.access_token = auth.authenticate()
```

Comment:
I think that `swagger-codegen` likes to call it `access_token`. No particular preference on naming from my side though!

Reply:
I'll see if I can change that, as it's a bit disingenuous and confusing, especially given the docs referring to an `id_token` :slightly_smiling_face:

Reply:
Fixed and have tested it locally with the example from the README

Reply:
Given that this may be already in use, would it be alright to keep using both fields? Mainly as to not disrupt users' existing scripts after the upgrade. If we want to, we can add a "soft-deprecation notice", similar to the one in use for the `predict(username, password)` method.

Reply:
Done in commit 5307221a4f32f69a3c4eeb69f8fd4fa6a52f7c05.

I've tested this locally in IPython with:
```python
In [1]: from seldon_deploy_sdk import Configuration
In [2]: config = Configuration()
In [3]: config.id_token = 'hello'
In [4]: config.access_token = 'world'
DEPRECATED!! Future versions of seldon_deploy_sdk will ignore the access_token field. Use the id_token field instead.
In [5]: config.access_token
Out[5]: 'world'
In [6]: config.id_token
Out[6]: 'world'
```